client/components/mobile-phone-verification/mobile-phone-verification.js

```diff
@@ -171,6 +174,39 @@ export default class MobilePhoneVerification extends React.Component {
       });
   }
 
+  async activePhoneToken() {
+    const {orgSlug, language, userData} = this.props;
+    const {errors} = this.state;
+    const url = mobilePhoneTokenStatusUrl(orgSlug);
+    return axios({
+      method: "get",
+      headers: {
+        "content-type": "application/x-www-form-urlencoded",
+        "accept-language": getLanguageHeaders(language),
+        Authorization: `Bearer ${userData.auth_token}`,
+      },
+      url,
+    })
+      .then((data) => data.active)
+      .catch((error) => {
+        if (error.response && error.response.status === 404) {
+          // This is kept for backward compatibility with older versions of OpenWISP RADIUS
+          // that does not have API endpoint for checking phone token status.
+          return false;
+        }
```

Comment:
The backend was already returning 404 response when the API endpoint is not present in OpenWISP RADIUS. I added this handling in the frontend.